grim.go

```diff
@@ -118,7 +119,11 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 			return grimErrorf("error extracting hook from message: %v", err)
 		}
 
-		if !(hook.EventName == "push" || hook.EventName == "pull_request" && (hook.Action == "opened" || hook.Action == "reopened" || hook.Action == "synchronize")) {
+		if !repoIsAlive(hook) {
```

Comment:
Doesnt make sense to return an error in this branch and then check the opposite in the other branch.  I assume returning an error is a mistake.